velox/expression/VectorWriters.h

```diff
@@ -715,10 +710,10 @@ struct VectorWriter<DynamicRow, void> : public VectorWriterBase {
 
   void ensureSize(size_t size) override {
     if (size > rowVector_->size()) {
-      rowVector_->resize(size, /*setNotNull*/ false);
       for (int i = 0; i < writer_.childrenCount_; ++i) {
```

Comment:
should we mention the same reason here as L280, if the same applies here ?

Reply:
So just add the comment. Will do.